extensions/linkedin/0.1.0/schema.json

```
@@ -0,0 +1,11 @@
+{
+    "type": "string",
+    "pattern": "^https://(www\\.)?linkedin\\.com.*$",
```

IMHO Doesn't matter if users write `http` or rather `https`. Both are valid. It is linkedin who should decide if redirect will happen or not.

```suggestion
    "pattern": "^http(s)?://(www\\.)?linkedin\\.com.*$",
```